docker/exporters/Dockerfile

```diff
@@ -25,16 +29,21 @@ RUN cd cmsweb-exporters && \
     cp process_monitor.sh /build
 
 # Adding apache exporter
-
-RUN wget https://github.com/Lusitaniae/apache_exporter/releases/download/v0.9.0/apache_exporter-0.9.0.linux-amd64.tar.gz && tar -xvzf apache_exporter-0.9.0.linux-amd64.tar.gz --directory /data/gopath && cp /data/gopath/apache_exporter-0.9.0.linux-amd64/apache_exporter /build
+#        https://github.com/Lusitaniae/apache_exporter/releases/download/v1.0.10/apache_exporter-1.0.10.linux-amd64.tar.gz
+RUN wget https://github.com/Lusitaniae/apache_exporter/releases/download/v${APACHEEXP_VER}/apache_exporter-${APACHEEXP_VER}.linux-amd64.tar.gz && \
+  tar -xvzf apache_exporter-${APACHEEXP_VER}.linux-amd64.tar.gz --directory /data/gopath && \
+  cp /data/gopath/apache_exporter-${APACHEEXP_VER}.linux-amd64/apache_exporter /build
 
 # build mongo exporter
+# TODO: consider using an actively developed exporter, such as
+#       https://github.com/percona/mongodb_exporter
```

Comment:
For this I suggest you open CMSMONIT issue and ask Monitoring group to install and test it with existing dashboard. If it is compatible then we can easily switch otherwise we need a new dashboard which will use this exporter metrics.

Reply:
i see, but since we are not using mongodb_expoerter in our wmcore images, can we merge this PR, build a new exporter image and tag it with `-stable`, use it for https://github.com/dmwm/CMSKubernetes/pull/1611 , then build a new tag with the new `percona/mongodb_exporter` after cmsmonit team finish their investigations?

Reply:
as I suggested please open up new CMSMON ticket and link it here for cross references, then we can proceed with merge and stable images.

Reply:
I opened the ticket [1] and added it to the comments in the dockerfile. 

> if you claim we (WM) group not using particular exporter it does not mean it is not in use.

I know, and this is why i proposed a change instead of just removing removing or changing it :)


[1] https://its.cern.ch/jira/projects/CMSMONIT/issues/CMSMONIT-650?filter=allissues